### PR TITLE
fix balky nei handlers

### PR DESCRIPTION
**Pull request by Glease**

### CODE REVIEW

File: src/main/java/com/github/bartimaeusnek/bartworks/neiHandler/BW_NEI_BioVatHandler.java [hunk 1]
```diff
@@ -107,23 +109,31 @@ public void drawExtras(int aRecipeIndex) {
...
+        } else {
+            super.loadUsageRecipes(aResult);
+        }
     }
 
     @Override
     public void loadCraftingRecipes(ItemStack aResult) {
-        if (aResult == null || !(aResult.getItem() instanceof LabParts && aResult.getItemDamage() < 3)) {
+        if (aResult != null && aResult.getItem() instanceof LabParts && aResult.getItemDamage() < 3 && aResult.getTagCompound() != null) {
+            loadLabPartRecipes(aResult);
```
Comment (miozune) [inline src/main/java/com/github/bartimaeusnek/bartworks/neiHandler/BW_NEI_BioVatHandler.java:137]: Is it intended that finding crafting recipe for petri dish shows recipes where dishes are actually inputs? (I haven't used Vat, if this is pointless you can just ignore me)

**Review by miozune — COMMENTED** [review]
oversight? https://github.com/GTNewHorizons/bartworks/blob/93e9f2206ea4a0ec4ec6787555fa37f2dd2cb154/src/main/java/com/github/bartimaeusnek/bartworks/neiHandler/BW_NEI_BioVatHandler.java#L47

File: src/main/java/com/github/bartimaeusnek/bartworks/neiHandler/BW_NEI_BioVatHandler.java (hunk 1 above)
Comment (Glease) [reply src/main/java/com/github/bartimaeusnek/bartworks/neiHandler/BW_NEI_BioVatHandler.java:137]: While it's looking in inputs list, what it's really looking for is the special slot item. I honestly don't know if bio vat might mutate that tbh.

**Review by miozune — APPROVED** [review]
Ok for now, other than the transferRect leftover

**Pull request merged**